### PR TITLE
:sparkles: Allows to provide ExtraConfig for Ironic conf

### PR DESCRIPTION
**Pull request by winiciusallan**
Closes: #87

### CODE REVIEW

File: api/v1alpha1/ironic_types.go [hunk 1]
```diff
@@ -194,6 +194,15 @@ type Images struct {
 	Keepalived string `json:"keepalived,omitempty"`
 }
 
+type ExtraConfig struct {
```
Comment (iurygregory) [inline api/v1alpha1/ironic_types.go:197]: I would probably add docs here  
// ExtraConfig defines ....

**Review by Rozzii — COMMENTED** [review]
/lgtm

File: api/v1alpha1/ironic_types.go [hunk 2]
```diff
@@ -194,6 +194,15 @@ type Images struct {
...
+
+	Group string `json:"group,omitempty`
```
Comment (tuminoid) [inline api/v1alpha1/ironic_types.go:199]: Add docs to each of these please.

Comment (winiciusallan) [reply api/v1alpha1/ironic_types.go:204]: Done!

**Comment by iurygregory** [comment]
lgtm

**Review by Rozzii — COMMENTED** [review]
/lgtm

**Review by tuminoid — COMMENTED** [review]
/approve

**Pull request merged**